diambra/arena/utils/controller.py

```diff
@@ -169,27 +167,19 @@ def save_device_configuration(self):
         cfg_file_dict_list.append(cfg_dict_to_save)
 
         # Open file (new or overwrite previous one)
-        cfg_file = open(self.device_config_file_name, "wb")
+        cfg_file = open(self.device_config_file_path, "wb")
         pickle.dump(cfg_file_dict_list, cfg_file)
         cfg_file.close()
 
     # Load all devices configuration
     def load_all_device_configurations(self):
 
-        try:
-            cfg_file = open(self.device_config_file_name, "rb")
-
-            # Load Pickle Dict
-            cfg_file_dict_list = pickle.load(cfg_file)
-
-            cfg_file.close()
-
-        except OSError:
-            print("No device configuration file found in: {}".format(os.path.join(home_dir, '.diambra/config/')))
-            config_file_folder = os.path.join(home_dir, '.diambra/')
-            os.makedirs(config_file_folder, exist_ok=True)
-            config_file_folder = os.path.join(home_dir, '.diambra/config/')
-            os.makedirs(config_file_folder, exist_ok=True)
+        if os.path.exists(self.device_config_file_path):
+            with open(self.device_config_file_path, "rb") as cfg_file:
+                cfg_file_dict_list = pickle.load(cfg_file)
+        else:
+            print("No device configuration file found in: {}".format(os.path.dirname(self.device_config_file_path)))
```

Comment:
Not necessarily for now but we should use the logger for these things

Reply:
Right, sure, done!